lib/chef/provisioning/driver.rb

```diff
@@ -102,11 +102,11 @@ def driver_options
     # gears moving, but the job doesn't need to be done :)
     #
     # @param [Chef::Provisioning::ActionHandler] action_handler The action_handler object that is calling this method
-    # @param [Chef::Provisioning::MachineSpec] machine_spec A machine specification representing this machine.
+    # @param [Chef::Provisioning::ManagedEntry] machine_spec A machine specification representing this machine.
     # @param [Hash] machine_options A set of options representing the desired options when
```

Comment:
Yep, ManagedEntry is growing on me looking at the changes. Feels like I can understand it better than MachineSpec